app/src/main/res/drawable-hdpi/button_bg_gray.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" android:padding="10dp">
+
+    <solid android:color="@color/white"/>
+    <stroke android:color="@color/bordercolor"
+        android:width="1dp"/>
+    <corners
+        android:radius="10dp"
+
+        />
```

Comment:
make dimens value and use them in all the xml file either layout or drawable

Reply:
These drawables are imported from another project for demo purposes only.
Please ignore changes like these for now.